language-server/src/json-instance-from-json.js

```diff
@@ -0,0 +1,261 @@
+import { parseTree } from "jsonc-parser";
+import * as JsonPointer from "@hyperjump/json-pointer";
+import { getKeywordId } from "@hyperjump/json-schema/experimental";
+import { drop, find, head, some } from "@hyperjump/pact";
+import { toAbsoluteUri } from "./util.js";
+
+export class JsoncInstanceFromJSON {
```

Comment:
There doesn't look like there's a good reason to create a new Instance class and the duplication is unacceptable. It looks like all you really needed was to add your `fromJSON` static function to the existing `JsoncInstance` class.